spec/controllers/application_submissions_spec.rb

```diff
@@ -395,6 +395,16 @@
         expect(response).not_to render_template 'show'
       end
     end
+    context 'record does not belong in the directory' do
```

Comment:
This `context` description is a little confusing. The context here is that there is no current user at all. It doesn't really matter what the state of the record is. Maybe `'no current user'` or something to that effect?